### DISCUSSION TITLE
Add tabs function

### DISCUSSION THREAD

**Discussion by noraa-junker** [community/community#4179] [Mobile] [answered]
I would love to see some sort of tab feature. 

Then I could have one tab with an issue open and one with the overvoew of issues of this repositorys. Then I could search in tab 2 if the issue in tab 1 was already proposed. 

That's my normal workflow, but that's currently only possible in a browser.

For a good example for tabs in an app see the official Wikipedia App

**Comment by alcere** [comment, marked as answer]
Hi! Thanks for your feedback! I shared your idea with the design team for evaluation. Please, keep the feedback coming!

**Reply by noraa-junker** [reply]
Thank you
